soc/renesas/ra/soc.yml

```diff
@@ -1,7 +1,7 @@
 family:
   - name: renesas_ra
     series:
-      - name: ra4m1
+      - name: ra4m1_arduino
```

Comment:
where is this changed in the Kconfig files?

Reply:
Sorry for the redundant change, this is a mistake. I removed this change. Thanks